.github/workflows/tests.yml

```diff
@@ -10,21 +10,18 @@ env:
 jobs:
   build:
 
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
     steps:
     - uses: actions/checkout@v1
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v1
       with:
-        python-version: 3.8
+        python-version: 3.12
     - name: Checkout submodules
       run: git submodule update --init --recursive
 
     - name: Setup
-      run: |
-        python -m pip install --upgrade pip
-        pip install -r requirements_dev.txt
-        git submodule update --init --recursive
```

Comment:
Does this mean the tests don't need the `360-ds` submodule?

Reply:
Oh the workflow succeeds doesn't it, so I guess not

Reply:
yes nothing should require that submodule as it is only there for when one wants to update the compiled css from `360-ds`.